synapse/storage/data_stores/main/events_worker.py

```diff
@@ -189,21 +190,23 @@ def get_events(
     @defer.inlineCallbacks
     def get_events_as_list(
         self,
-        event_ids,
-        check_redacted=True,
-        get_prev_content=False,
-        allow_rejected=False,
+        event_ids,  # type: List[Any]
+        check_redacted=True,  # type: bool
```

Comment:
```suggestion
        check_redacted: bool=True,
```

(or whatever black makes of that)

likewise below